### ISSUE TITLE
API Groups are no longer selectable in Swagger UI when using Spring MVC

### ISSUE THREAD

**Issue by adrian-haenni** [springdoc/springdoc-openapi#1344]
**Describe the bug**
When upgrading in a Spring MVC configuration from version 1.5.10 to 1.5.12, the Swagger UI does not show the "Select a definition" drop down for GroupedOpenApi Beans anymore. I also noticed that the generated api-docs/swagger-config JSON is missing entries for the URLs of GroupedOpenApi's. When I enter the path of an API group directly into the "Explore" textfield, then it will be shown correctly. However, I expect that the "Select a definition" drop down should be shown instead of "Explore" textfield.

**To Reproduce**
I have the following configurations:

SwaggerOpenApiConfig.java

```
@Configuration
@EnableWebMvc
@ComponentScan(basePackages = {"org.springdoc"})
@PropertySource("classpath:springdoc.properties")
@Import({org.springdoc.core.SpringDocConfiguration.class,
        org.springdoc.webmvc.core.SpringDocWebMvcConfiguration.class,
        org.springdoc.webmvc.core.MultipleOpenApiSupportConfiguration.class,
        org.springdoc.webmvc.ui.SwaggerConfig.class,
        org.springdoc.core.SwaggerUiConfigProperties.class,
        org.springdoc.core.SwaggerUiOAuthProperties.class,
        org.springdoc.core.SpringDocConfigProperties.class,
        org.springframework.boot.autoconfigure.jackson.JacksonAutoConfiguration.class})
public class SwaggerOpenApiConfig {

    @Autowired
    OpenApiCustomiser openApiSortCustomiser;

    // FULL API

    @Bean
    public OpenAPI integrationAPI() {
        return new OpenAPI()
                .info(new Info().title("Integration API")
                        .description("The full REST interface for integration tasks")
                        .version("v1.0.0"))
                .components(new Components()
                        .addSecuritySchemes("api_auth",
                                new SecurityScheme()
                                        .type(SecurityScheme.Type.HTTP)
                                        .name("api_auth")
                                        .scheme("basic")
                                        .in(SecurityScheme.In.HEADER)
                                        .name("Authorization")))
                .addSecurityItem(new SecurityRequirement().addList("api_auth", Arrays.asList("read", "write")));
    }

    // API GROUPS

    @Bean
    public GroupedOpenApi groupFullOpenApi() {
        String[] packagesToScan = {"com.some.integration"};
        return GroupedOpenApi.builder()
                .packagesToScan(packagesToScan)
                .group("")
                .addOpenApiCustomiser(openApiSortCustomiser)
                .build();
    }

    @Bean
    public GroupedOpenApi groupAvailabilityOpenApi() {
        String[] packagesToScan = {"com.some.integration.availabilitychecker"};
        return GroupedOpenApi.builder()
                .packagesToScan(packagesToScan)
                .group("availability-checker")
                .addOpenApiCustomiser(openApiSortCustomiser)
                .build();
    }

    @Bean
    public OpenApiCustomiser sortTagsAlphabetically() {
        return openApi -> openApi.setTags(openApi.getTags()
                .stream()
                .sorted(Comparator.comparing(tag -> StringUtils.stripAccents(tag.getName())))
                .collect(Collectors.toList()));
    }
```

springdoc.properties

```
springdoc.swagger-ui.disable-swagger-default-url=true
springdoc.api-docs.groups.enabled=true
```

**Expected behavior**
I expect that the "Select a definition" drop down should be shown instead of "Explore" textfield.

**Screenshots**
Before Upgrade:
![Screenshot from 2021-11-22 15-10-39](https://user-images.githubusercontent.com/9133952/142876444-fbfa22b4-5b9e-4cee-8367-2ab489ed612e.png)
After Upgrade:
![Screenshot from 2021-11-22 15-08-47](https://user-images.githubusercontent.com/9133952/142876446-a7cab009-dd2e-46b3-a3c2-4b84a7df8e2f.png)

**Additional context**
I noticed that if i define the groups in ```springdoc.properties``` it works for the 1.5.12 version, e.g. with
```
springdoc.group-configs[1].group=availability-checker
springdoc.group-configs[1].packages-to-scan=com.some.integration.availabilitychecker
```

However, it is still an issue if it does not work with java config.

**Comment by bnasslahsen** [comment]
@adrian-haenni, 

The procedure is [all documented here](https://springdoc.org/#how-to-integrate-open-api-3-with-spring-project-not-spring-boot).
If you believe you are facing bug, please make sure you provide a  [Minimal, Reproducible Example - with HelloController ](https://stackoverflow.com/help/minimal-reproducible-example)that reproduces the problem.

This ticket will be closed, but can be reopened if any relevant information is provided.

**Comment by adrian-haenni** [comment]
@bnasslahsen

[SpringMvcApiGroupExample.zip](https://github.com/springdoc/springdoc-openapi/files/7593968/SpringMvcApiGroupExample.zip)

I was able to reproduce the issue with the attached minimal example. Just switch the springdoc version in the pom file.

**Comment by bnasslahsen** [comment]
@adrian-haenni, 

Can you test using the[ latest SNAPSHOT](https://springdoc.org/#how-can-i-use-the-last-springdoc-openapi-snapshot) ?
This issue seems to be fixed using: `1.5.13-SNAPSHOT`.

**Comment by adrian-haenni** [comment]
@bnasslahsen 

Issue is gone with 1.5.13-SNAPSHOT. Thanks!
